src/Illuminate/Support/helpers.php

```diff
@@ -142,7 +142,13 @@ function data_get($target, $key, $default = null)
 
         $key = is_array($key) ? $key : explode('.', $key);
 
-        while (! is_null($segment = array_shift($key))) {
+        foreach ($key as $i => $segment) {
+            unset($key[$i]);
+
+            if ($segment === null) {
```

Comment:
there was a PR awhile ago that switched `$var === null` to `is_null($var)`.

https://github.com/laravel/framework/pull/15655/files

Might want to switch this line to maintain consistency.

Reply:
This PR is in feature parody with the current 7.x state.

Reply:
I agree the behavior is the same, but to maintain style consistency, we should switch `$segment === null` to `is_null($segment)`.

Also, FYI. *Parity*, not *parody*.

Reply:
Done